samples/snippets/csharp/VS_Snippets_Remoting/Classic HttpWebRequest.ConnectionGroupName Example/CS/source.cs

```diff
@@ -12,8 +12,8 @@ public static int Main(String[] args)
  {
 // <Snippet1>
   // Create a secure group name.
-  SHA1Managed Sha1 = new SHA1Managed();
-  Byte[] updHash = Sha1.ComputeHash(Encoding.UTF8.GetBytes("username" + "password" +  "domain"));
+  SHA256 Sha256 = SHA256.Create();
+  Byte[] updHash = Sha256.ComputeHash(Encoding.UTF8.GetBytes("username" + "password" +  "domain"));
   String secureGroupName = Encoding.Default.GetString(updHash);

```

Comment:
Not really the problem at hand, but: I don't know that this is going to do anything useful.  The hash may contain a zero-byte, which will make the string effectively terminate early.

It's also weird that the "secure group name" is a hash of static data.  Might want to get networking to review this sample to see what it's supposed to be doing.

Reply:
This is for HttpWebRequest, which has an **Important** alert box warning that "We don't recommend that you use HttpWebRequest for new development. Instead, use the System.Net.Http.HttpClient class." The example is for [ConnectionGroupName](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpwebrequest.connectiongroupname?view=netcore-3.1) property. It may not be worth someone's time to investigate and I should just revert the changes on these files(?)

Reply:
Yeah, reverting the edit and changing it to the "don't use SHA1" comment may be the short-term best.